src/overlays/actors/ovl_En_Zo/z_en_zo.h

```diff
@@ -7,12 +7,33 @@ struct EnZo;
 
 typedef void (*EnZoActionFunc)(struct EnZo*, GlobalContext*);
 
+#define ENZO_GET_PATH(thisx) (((thisx)->params & 0x7E00) >> 9)
+#define ENZO_NO_PATH(thisx) (ENZO_GET_PATH(thisx) != 0x3F)
```

Comment:
Its probably better to just use `#define ENZO_NO_PATH 0x3F`, then you can use `func_8013D648(globalCtx, ENZO_GET_PATH(&this->actor), ENZO_NO_PATH )` in Init as well, which makes sense in the context of SubS work in progress.

Reply:
Yeah, sorry, that's what I meant.

Reply:
Ah ok, sorry about that.